sources/packages/web/src/services/http/dto/Application.dto.ts

```diff
@@ -86,6 +86,18 @@ export interface ApplicationDataAPIOutDTO extends ApplicationBaseAPIOutDTO {
   submittedDate?: Date;
 }
 
+/**
+ * DTO for application data
+ */
+export interface ApplicationSupplementalDataAPIOutDTO
+  extends ApplicationBaseAPIOutDTO {
+  studentFullName: string;
+  applicationOfferingIntensity: OfferingIntensity;
+  applicationStartDate: string;
+  applicationEndDate: string;
+  applicationInstitutionName: string;
```

Comment:
Following the API DTO, these fields must be optional.

Reply:
I changed to API DTO to match this version.